internal/client/influxdb/influxdb.go

```diff
@@ -0,0 +1,51 @@
+package influxdb
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/influxdata/influxdb/client/v2"
+	"github.com/mainflux/mainflux/internal/env"
+	"github.com/mainflux/mainflux/pkg/errors"
+)
+
+var (
+	errConnect = errors.New("failed to create InfluxDB client")
+	errConfig  = errors.New("failed to load  InfluxDB client configuration from environment variable")
+)
+
+type Config struct {
+	Protocol           string        `env:"PROTOCOL"              envDefault:"http"`
+	Host               string        `env:"HOST"                  envDefault:"localhost"`
+	Port               string        `env:"PORT"                  envDefault:"8086"`
+	Username           string        `env:"ADMIN_USER"            envDefault:"mainflux"`
+	Password           string        `env:"ADMIN_PASSWORD"        envDefault:"mainflux"`
+	DbName             string        `env:"DB"                    envDefault:"mainflux"`
+	UserAgent          string        `env:"USER_AGENT"            envDefault:"InfluxDBClient"`
+	Timeout            time.Duration `env:"TIMEOUT" `
```

Comment:
No default?

Reply:
In InfluxDB there is no timeout if it is not set, so only didn't set any default 

Reference here: https://github.com/mainflux/mainflux/blob/master/vendor/github.com/influxdata/influxdb/client/v2/client.go#L39-L40